.github/workflows/build-cachelib-debian-10.yml

```diff
@@ -52,6 +52,9 @@ jobs:
           g++ - || true
       - name: "checkout sources"
         uses: actions/checkout@v2
+      - name: "Add Git safe directory"
+        # Workaround for Docker image bug (GitHub issue #199).
+        run: git config --system --add safe.directory /__w/CacheLib/CacheLib
```

Comment:
Note that this will fail for forks if the fork's name is not CacheLib. Can be resolved by using `/__w/${{github.event.repository.name}}/${{github.event.repository.name}}`, but I worried this was overthinking it especially for a temporary workaround + [some sources](https://stackoverflow.com/questions/62803531/repository-name-as-a-github-action-environment-variable#comment120510045_62805013) say event is unavailable for scheduled workflow pushes.

Reply:
Oh, not familiar with this, but can we use the [github variable](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) `$GITHUB_WORKSPACE`? Not sure what is difference against [github context](https://docs.github.com/en/actions/learn-github-actions/contexts) `${{ github.workspace }}`

Reply:
Ooh, nice find, that's better. I've changed it. Sounds like it's the same, from the page you linked to:

> Most of the default variables have a corresponding, and similarly named, context property